sdk/resourcemanager/azure-resourcemanager-test/src/main/java/com/azure/resourcemanager/test/ResourceManagerTestProxyTestBase.java

```diff
@@ -265,34 +258,17 @@ protected void beforeTest() {
                 addSanitizers();
             }
         } else {
-            if (System.getenv(AZURE_AUTH_LOCATION) != null) { // Record mode
-                final File credFile = new File(System.getenv(AZURE_AUTH_LOCATION));
-                try {
-                    testAuthFile = AuthFile.parse(credFile);
-                } catch (IOException e) {
-                    throw LOGGER.logExceptionAsError(new RuntimeException("Cannot parse auth file. Please check file format.", e));
-                }
-                credential = testAuthFile.getCredential();
-                testProfile = new AzureProfile(testAuthFile.getTenantId(), testAuthFile.getSubscriptionId(), testAuthFile.getEnvironment());
-            } else {
-                Configuration configuration = Configuration.getGlobalConfiguration();
-                String clientId = configuration.get(Configuration.PROPERTY_AZURE_CLIENT_ID);
-                String tenantId = configuration.get(Configuration.PROPERTY_AZURE_TENANT_ID);
-                String clientSecret = configuration.get(Configuration.PROPERTY_AZURE_CLIENT_SECRET);
-                String subscriptionId = configuration.get(Configuration.PROPERTY_AZURE_SUBSCRIPTION_ID);
-                if (clientId == null || tenantId == null || clientSecret == null || subscriptionId == null) {
-                    throw LOGGER.logExceptionAsError(
-                        new IllegalArgumentException("When running tests in record mode either 'AZURE_AUTH_LOCATION' or 'AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET and AZURE_SUBSCRIPTION_ID' needs to be set"));
-                }
-
-                credential = new ClientSecretCredentialBuilder()
-                    .tenantId(tenantId)
-                    .clientId(clientId)
-                    .clientSecret(clientSecret)
-                    .authorityHost(AzureEnvironment.AZURE.getActiveDirectoryEndpoint())
-                    .build();
-                testProfile = new AzureProfile(tenantId, subscriptionId, AzureEnvironment.AZURE);
-            }
+            Configuration configuration = Configuration.getGlobalConfiguration();
+            String tenantId = Objects.requireNonNull(
+                configuration.get(Configuration.PROPERTY_AZURE_TENANT_ID),
+                "'AZURE_TENANT_ID' environment variable cannot be null.");
+            String subscriptionId = Objects.requireNonNull(
+                configuration.get(Configuration.PROPERTY_AZURE_SUBSCRIPTION_ID),
+                "'AZURE_SUBSCRIPTION_ID' environment variable cannot be null.");
+            credential = new DefaultAzureCredentialBuilder()
+                .authorityHost(AzureEnvironment.AZURE.getActiveDirectoryEndpoint())
+                .build();
```

Comment:
I think it can not working,
the `DefaultAzureCredential` still requires `tanentId`, `clientId` and `clientSecret`.